From: Haifeng Xu <haifeng.xu@shopee.com>
Subject: mm/memory_hotplug: remove reset_node_managed_pages() in hotadd_init_pgdat()
Date: Thu, 1 Jun 2023 16:26:25 +0000

managed pages has already been set to 0 in free_area_init_core_hotplug(),
so it's pointless to reset again.

Link: https://lkml.kernel.org/r/20230601162626.1030-1-haifeng.xu@shopee.com
Signed-off-by: Haifeng Xu <haifeng.xu@shopee.com>
Reviewed-by: Andrew Morton <akpm@linux-foudation.org>
Cc: David Hildenbrand <david@redhat.com>
Cc: Haifeng Xu <haifeng.xu@shopee.com>
Cc: Michal Hocko <mhocko@kernel.org>
Cc: Mike Rapoport (IBM) <rppt@kernel.org>
Cc: Oscar Salvador <osalvador@suse.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/memory_hotplug.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/memory_hotplug.c~mm-memory_hotplug-remove-reset_node_managed_pages-in-hotadd_init_pgdat
+++ a/mm/memory_hotplug.c
@@ -1210,7 +1210,6 @@ static pg_data_t __ref *hotadd_init_pgda
 	 * online_pages() and offline_pages().
 	 * TODO: should be in free_area_init_core_hotplug?
 	 */
-	reset_node_managed_pages(pgdat);
 	reset_node_present_pages(pgdat);
 
 	return pgdat;
_
